libs/megadrive-graphics/default_font.py

```diff
@@ -0,0 +1,89 @@
+import math
+import os.path
+from typing import TextIO
+
+from PIL import Image, ImageDraw, ImageFont
+# Regarding Pillow license: the Historical Permission Notice and Disclaimer
+# https://github.com/python-pillow/Pillow/blob/master/LICENSE
+# Since we re-use the "default font" from Pillow, we thought it prudent to mention the license and to declare that we
+# believe it's compatible with this project's MIT license.
+
+# chars 1-31 are SOH, STX, ... TAB, LF etc., are non-drawable. char 128 is DEL
+# chr(0) is the NUL char, so we'll just render this as a black tile
+drawable_chars = [0]
+drawable_chars.extend(list(range(32, 128)))
+
+boilerplate = """// This file was generated from the Python ascii_rs_generator script, please don't edit directly
+use megadrive_sys::vdp::Tile;
+use megadrive_sys::vdp::SpriteSize::Size1x1;
+use crate::font::Font;
+
+pub static DEFAULT_FONT_1X1: Font = Font { tile_data: TILE_DATA, sprite_size: Size1x1, start_index: 1 };
+
+pub static TILE_DATA: &'static [Tile] = &[
+"""
+
+
+def generate_image_arrays():
+    output_path = 'src/default_ascii.rs'
```

Comment:
```suggestion
    output_path = os.path.join(os.environ['OUT_DIR'], 'default_ascii.rs')
```
Cargo build scripts should write to `$OUT_DIR`. The output shouldn't be checked in any more (it'll be regenerated if needed on build).

Reply:
Ah I see. Fixed in fb88f9416edcdafd273462b96a2f6d3f9db5064f and 50c230c1ed40b1b2009804f2b219cc51d06ea223

Reply:
Just slighty fiddly to get the hang of the `build.rs` system, but it looks like it's getting pretty Rust-idiomatic